website/_data/roadmap.yml

```diff
@@ -1,37 +1,51 @@
 - status: upcoming
-  title: Smart contract and Decentralized Storage Platform
-  date: Q1 2025
+  title: Pactus Onchain Governance
+  date: Q1 2026
+
+- status: upcoming
+  title: Decentralized Domain Name System
+  date: Q4 2025
+
+- status: upcoming
+  title: Decentralized Password Manger
+  date: Q4 2025
 
 - status: upcoming
   title: Browser Extension
-  date: Q1 2025
+  date: Q3 2025
+
+- status: upcoming
+  title: Pactus Virtual Machine (PVM)
```

Comment:
PVM is already used in Polkadot, we may need a new name. also, We can use Pactus beside the VM when we designed the VM ourselves I believe.

Reply:
OK, we discuss for a name, lets keep this name for now.